Smartpad-iOS/Smartpad-iOS/Controller/MainViewController.swift

```diff
@@ -59,18 +59,19 @@ class MainViewController: UIViewController {
 //    }
 
     @IBAction func singleTapRecognizer(_ recognizer: UITapGestureRecognizer) {
-//        hapticManager?.playTouchdown()
+        hapticManager?.playTouchRelease()
```

Comment:
It would be better if we can get state events from single tap,

Basically,

on tap start: Send push down haptic
on tap end: Send release haptic

Reply:
actually I like the idea of having a new gesture recognizer just for the haptic feedback, though I don't know if that is possible. 

I'm also not sure if UITapGesture has start and end states similar to that of UIPanGesture. It seems like their action methods only gets triggered upon release. 

I'll look into these.

Reply:
Make sense, either approach should work, I'd also favour having a new gesture recognizer for just haptic feedback